### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by AsumVictor**
# Milestone 2: Components
In this PR, I have components of the calculator

 ## Type of change
 * [ ] Created new directory `components` in the `src` folder
 * [ ] Created `Button.js` to add functionality to the buttons in the calculator
 * [ ] Created `Screen.js` to add the functionality of the screen of the calculator
 * [ ] Created `Calculator.js` to add the functionality  the calculator
 
 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linter error in my code
 * [ ]  My changes generate no new warnings

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <Button value="6" />
+      <Button value="-" classextend="bg-orange-500" />
+      <Button value="1" />
+      <Button value="2" />
+      <Button value="3" />
+      <Button value="+" classextend="bg-orange-500" />
+      <Button value="0" classextend="col-span-2" />
+      <Button value="." />
+      <Button value="=" classextend="bg-orange-500" />
+    </div>
```
Comment (AbayomiOlaoye) [inline src/components/Calculator.js]: |Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

- [x] Good job with the job well done 
:clap: but I think we have a design to match here. Kindly apply appropriate styling/colors as seen in the wireframe. It looks like the operands or operation buttons have a different color on them, maybe you should try this: `#de935f`

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @AsumVictor :wave: :wave:  

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Great use of functional component :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (AsumVictor) [reply src/components/Calculator.js]: Thank you for the notice! I have resolved that 👏

**Review by OLIPLICHE — APPROVED** [review]
Hi @AsumVictor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the change requested by the previous review👏
- the calculator matches the given design👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsumVictor** [comment]
Thank you very much!

**Pull request merged**